ql/src/java/org/apache/hadoop/hive/ql/exec/repl/OptimisedBootstrapUtils.java

```diff
@@ -87,20 +88,20 @@ public static boolean checkFileExists(Path dumpPath, HiveConf conf, String fileN
   }
 
   /**
-   * Gets the event id from the event ack file
+   * Gets the source & target event id  from the event ack file
    * @param dumpPath the dump path
    * @param conf the hive configuration
    * @return the event id from file.
    * @throws IOException
    */
-  public static String getEventIdFromFile(Path dumpPath, HiveConf conf) throws IOException {
+  public static String[] getEventIdFromFile(Path dumpPath, HiveConf conf) throws IOException {
```

Comment:
When does tis. return more than one values?

Reply:
It would be always 2, one from the source & one with respect to the target.
The javadoc says that above.
```
Gets the source & target event id  from the event ack file
```